backend/app/services/adapters/task_kinds/queries.py

```
@@ -0,0 +1,979 @@
+        result = build_lite_task_list(db, paginated_tasks, user_id, member_task_ids)
+
+        return result, len(valid_tasks)
+
+    def get_user_personal_tasks_lite(
+        self,
+        db: Session,
+        *,
+        user_id: int,
+        skip: int = 0,
+        limit: int = 50,
+        types: List[str] = None,
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)